testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1109,3 +1109,8 @@
 Then(/^I should see the text "(.*?)" in the (Operating System|Architecture|Channel Label) field/) do |text, field|
   page.has_field?(text, with: field)
 end
+
+Then(/^I should see the current time as (Start Time|End Time)/) do |timestamp|
+  now = Time.now
+  page.has_field?(now.strftime('%k:%M').to_s, with: timestamp)
```

Comment:
This has few votes for a flaky test :) 
I think somehow... we need to give a "range" for that check, as it depends at what moment in time you race `Time.now` it might not match with the text you would have in the WebUI.

Reply:
I thought about that aswell. 🤔 
It assumes you can switch to the page quickly enough to have this timestamp show up in the Start Time column...
Maybe add a check to pass this step when it's the current time + 60 seconds?

Reply:
> I thought about that aswell. thinking It assumes you can switch to the page quickly enough to have this timestamp show up in the Start Time column... Maybe add a check to pass this step when it's the current time + 60 seconds?

You can have the exact race condition 60 seconds later, the only way is giving a range, so you accept anything around +/- 5 seconds for instance.

Reply:
Ah alright I'll look into it :)